src/state/slices/preferencesSlice/preferencesSlice.ts

```diff
@@ -54,6 +55,7 @@ const initialState: Preferences = {
     WalletConnectToDapps: getConfig().REACT_APP_FEATURE_WALLET_CONNECT_TO_DAPPS,
     DashboardBreakdown: getConfig().REACT_APP_DASHBOARD_BREAKDOWN,
     FiatPopup: getConfig().REACT_APP_FEATURE_FIAT_POPUP,
+    EligibleEarn: getConfig().REACT_APP_FEATURE_ELIGIBLE_EARN,
```

Comment:
Add in `.env.base` as false and `.env.develop` and `.env.dev` as true, toggling via flags UI is deprecated-ish